From 8d550bbddd27af743d3527a52ced3fe909cbb25a Mon Sep 17 00:00:00 2001
From: Robin Burchell <robin+git@viroteck.net>
Date: Thu, 11 Jul 2013 11:32:20 +0000
Subject: [PATCH] Patch out docs build.

---
 Makefile.am  | 2 +-
 configure.ac | 4 ----
 2 files changed, 1 insertion(+), 5 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 022de5c..2d5936f 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -2,7 +2,7 @@ ACLOCAL_AMFLAGS = -I m4
 DISTCHECK_CONFIGURE_FLAGS = \
 	--enable-gtk-doc \
 	--enable-introspection=yes
-SUBDIRS = libsignon-glib docs tests
+SUBDIRS = libsignon-glib tests
 
 if ENABLE_PYTHON
 SUBDIRS += pygobject
diff --git a/configure.ac b/configure.ac
index 2397082..5a540de 100644
--- a/configure.ac
+++ b/configure.ac
@@ -28,8 +28,6 @@ AM_PROG_CC_C_O
 AC_HEADER_STDC
 AC_PROG_LIBTOOL
 
-GTK_DOC_CHECK([1.14], [--flavour no-tmpl])
-
 AC_ARG_ENABLE(cast-checks,  [  --disable-cast-checks   compile with GLIB cast checks disabled],[cchecks=${enableval}],cchecks=yes)
 if test "x$cchecks" = "xno"; then
     CFLAGS="$CFLAGS -DG_DISABLE_CAST_CHECKS"
@@ -82,8 +80,6 @@ AC_OUTPUT([
 	Makefile
 	libsignon-glib/Makefile
 	libsignon-glib.pc
-	docs/Makefile
-	docs/reference/Makefile
 	tests/Makefile
 	pygobject/Makefile
 ])
-- 
1.8.3-rc3

